spec/David_tic_tac_toe/validation_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../../lib/David_tic_tac_toe/validation'
+
+RSpec.describe Validation do
+  let(:input) { '12 , 12,a' }
+  it "should return false if input is not separated by a single comma" do
+    expect(subject.validate_input(input)).to eq false
+  end
+
+  it "should return false if input contains a non numeric character" do
+    expect(subject.validate_input('12,a')).to eq false
+  end
```

Comment:
I would have these two test cases in a context under `.validate_input`.

Reply:
It looks like you are missing test cases for `input.gsub!(/[[:space:]]/, '')`. If you don't have a test for it, then we don't know if the code works!

Reply:
> I would have these two test cases in a context under `.validate_input`.

1b8092c74aedfd9c1ae34fa91dee6692b08e5eb5